### PR TITLE
RestInParse 0.0.7

### PR DESCRIPTION
**Pull request by jmfayard**

### CODE REVIEW

File: restinparse-core/src/test/kotlin/com/github/jmfayard/ApiTests.kt [hunk 1]
```diff
@@ -58,6 +58,48 @@ class ApiTests : RxSpec() {
...
+
+      rxScenario("Real Booleans are True or False", zip) { triple ->
+        triple.debug("Real Booleans")
+
+        (triple.first + triple.second) shouldBe triple.third
+      }
+
+    }
+
+    feature("Testing not equsl") {
```
Comment (j-koenig) [inline restinparse-core/src/test/kotlin/com/github/jmfayard/ApiTests.kt]: typo

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 
-0.06
+
+RestInParse 0.07
+===
+
+- Implements `Parse.Query.count()`
+- New criteria `Parse.Query.isTrue()` and `Parse.Query.isNotTrue()` which search for either `false` `null` or `undefined`
+- New criteria `notEqualToInt()`, `notEqualToPtr`, `notEqualToString`
```
Comment (philiplb) [inline CHANGELOG.md:8]: Just a random idea: Jasmine has solved negations like this:
Parse.Query.isTrue()
Parse.Query.not.isTrue()
Parse.Query.equalToInt()
Parse.Query.not.equalToInt()
etc..
Maybe this is easier to implement and to use?

File: restinparse-core/src/main/java/com/github/jmfayard/ParseQuery.java [hunk 3]
```diff
@@ -18,7 +15,7 @@
     public final String className;
     public final Map<String, String> params;
 
-    ParseQuery(String className, Map<String, String> params, Map<String, Object> where, List<String> includes, int skip, int limit, String order) {
+    ParseQuery(String className, Map<String, String> params, Map<String, Object> where, List<String> includes, int skip, int limit, String order, int count) {
```
Comment (philiplb) [inline restinparse-core/src/main/java/com/github/jmfayard/ParseQuery.java:18]: Map<String, Object> where
->
Map<String, ?> where
or something implementing a not yet existing interface?
Map<String, ? extends RIPWhere> where

**Review by philiplb — APPROVED** [review]
Just added 1-2 random thoughts. :)

**Pull request merged**